### DISCUSSION TITLE
Environment variable for vcpkg location should be `VCPKG_ROOT`

### DISCUSSION THREAD

**Discussion by michaelbprice** [actions/virtual-environments#6146] [Ideas]
The vcpkg community, including the maintainers, have coalesced around using the environment variable `VCPKG_ROOT` to designate the location of the tool. The images for the GitHub0-hosted runners should migrate away from using `VCPKG_INSTALLATION_ROOT` to `VCPKG_ROOT` as doing so will simplify the use of vcpkg across varying tools and environments.

**Comment by miketimofeev** [comment]
Hey, Michael! 
Thanks for the heads up! I think it makes sense to keep both variables for some time to preserve backward compatibility. Do you have any objections?

**Reply by michaelbprice** [reply]
Yes, absolutely should keep both variables, with a clear indication in the documentation that `VCPKG_INSTALLATION_ROOT` is deprecated, and if possible make it clear when it will go away.
